check.go

```diff
@@ -109,25 +108,30 @@ func (c *C) stopNow() {
 // logger is a concurrency safe byte.Buffer
 type logger struct {
 	sync.Mutex
-	writer bytes.Buffer
+	buffer bytes.Buffer
+	output io.Writer
+	verbosity uint8
 }
 
 func (l *logger) Write(buf []byte) (int, error) {
 	l.Lock()
 	defer l.Unlock()
-	return l.writer.Write(buf)
+	if l.verbosity > 1 {
```

Comment:
I wouldn't do a 'greater than' comparison here, but an equal to the stream level (2). IMO in this case is clear that the verbosity is used for two different things, if it has one specific value the output goes through one variable and if not it goes through another one.